contributing/review-process.md

```diff
@@ -38,6 +38,40 @@ In The Falco Project, we strive to foster a positive and constructive environmen
 - When expressing your code style preferences, please prioritize specific style modifications that bring clear benefits to the project in terms of performance or maintainability. It is important to avoid excessive emphasis on personal style. This approach promotes inclusivity and respect for different coding styles, as long as they align with the project's best interests.
 - Please remember that in the realm of security, striking a balance between speed, caution, and code reorganization is crucial, considering that time is of the essence. This balanced approach allows us to effectively support the necessary capabilities for enhanced threat detection, performance, and scalability benefits within reasonable timelines.
 
+## How the Review Process Works
+
+ - The author submits a PR.
+
+- Automation ([@poiana](https://github.com/poiana)) suggests reviewers for the PR:
+
+  - Determine the set of [OWNERS](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#repository-ownership) files.
+  - Choose at least two suggested *reviewers*, trying to find a unique reviewer for every leaf OWNERS file, and request their reviews on the PR. 
+- Humans review the PR:
+   - Reviewers look for general code quality, correctness, style and follow [reviewers guideline](#guidelines-for-code-reviewers)
+   - Anyone in the determined set of OWNERS (including both `reviewers` and `approvers`) can act as a reviewer, with the exception of the individual who opened the PR.
+   - If the code changes look good to them, a reviewer types `/lgtm` in a PR comment or selects the _Approved_ checkbox and clicks the _Submit review_ button on the GitHub UI; if they change their mind, they can use `/lgtm cancel.`
```

Comment:
I believe we always need the git approve. It's not an or.

In addition plz change to  "either `/lgtm` or `/approve`" etc.

Reply:
Thanks @incertum for your review. Is it necessary for reviewers to also click the 'Submit review' button from the GitHub UI, or is '/lgtm' sufficient? I think selecting checkbox is must for approving changes only not for reviewing🤔.

Reply:
I see we want to separate the review from the actual approval. I think the review section then really can just be reviewing the PR and all of the lgtm and approve could be moved to the approval section? @leogr ?

Re approval: You need to check mark the git approve button and click the 'Submit review' button alongside either /lgtm or /approve.

Reply:
<img width="698" alt="image" src="https://github.com/falcosecurity/.github/assets/24579375/651b7ae9-cca0-4204-b9d8-7b8074eb5e65">